### ISSUE TITLE
SPIKE - UI backend improvements - Related articles

### ISSUE THREAD

**Issue by cdanfon** [MozillaFoundation/foundation.mozilla.org#9237]
## Describe the feature request

As an admin/editor Wagtail user adding supporting articles to a page ie: the PNI homepage, I want to be presented with only valid options instead of the whole website tree or to be given a cue of which path I need to follow to avoid me from navigating endless pages

## Screenshots / mock-ups

Attached video of the current behaviour

https://user-images.githubusercontent.com/102978346/184855960-91c1ba76-79d1-4af9-9763-aadef0fe3efd.mov

## Acceptance criteria

- [ ] As a Wagtail user is easy and clear for me to see which related article pages I can add

**Comment by cdanfon** [comment]
We should be able to enable this fairly easily as it is an 'out of the box' functionality with Wagtail.

**Comment by tbrlpld** [comment]
@cdanfon I think I might have made the comment that this should be easy to add, but I think this is not really the case. From the docs, we have the option to specify the page type, but we are already doing that.

https://docs.wagtail.org/en/stable/reference/pages/panels.html#pagechooserpanel

Apart from that, this might need a spike to explore what is possible.

**Comment by cdanfon** [comment]
Hey @tbrlpld 

Sorry, after looking into this further I think the ticket is too broad and would need to handle specific scenarios. 

For the example, for PNI in the exmple I gave, the way to improve the experience for an editor would be to only present 'buyers guide articles' instead of the whole site tree. The difference between the current behaviour:

- only article pages can be selected + the page chooser presents you with the whole site tree as the starting point

and the one I'm suggesting:

- only article pages can be selected + the page chooser presents you with the articles as the starting point (in the correct locale)


So basically all these steps would be avoided: 

https://www.loom.com/share/c7a459a3c3f84f41b3e8d43782409284

**Comment by tbrlpld** [comment]
It was not me, but there is a ticket for this in Wagtail: https://github.com/wagtail/wagtail/issues/3322

**Comment by tbrlpld** [comment]
Also: https://github.com/wagtail/wagtail/issues/2756

**Comment by cdanfon** [comment]
I've removed the grooming and discussions labels as this is now a SPIKE 

Timeboxed to 3 hours
